main.py

```diff
@@ -152,10 +152,17 @@ def check_configurations(args, pretrain_tasks, target_tasks):
 
     assert_for_log(
         args.transfer_paradigm in ["finetune", "frozen"],
-        "Transfer paradigm %s not supported!" % args.transfer_paradigm,
+        "Transfer paradigm %s not supported." % args.transfer_paradigm,
     )
 
     if args.do_pretrain:
+        task_directory, _, _ = check_for_previous_checkpoints(
+            args.run_dir, target_tasks, "target_train", load_model=1
+        )
+        if task_directory is not None and args.load_model:
+            log.warning(
+                "do_pretrain = 1 but there are currently checkpoints in the target train stage. If you mean to start running from target_train, set do_pretrain=0, do_target_train=1, and load_model=1."
```

Comment:
This looks like it could backfire. What if someone sets do_pretrain = 1, do_target_train = 1, launches a cluster job, and it crashes during target training? In that case, we should enter into the pretraining code, but then observe that the training job is done and move onto target task training without actually touching any data.

(If we keep this, be consistent about spacing around =.)

Reply:
Right, and we know the training job is done because of the stopped variable that we save in the training state.

Reply:
In that case,we can just remove this assert.